### PR TITLE
Add simple POD class to internalize keywords xxxDENT

### PR DESCRIPTION
**Pull request by joakim-hove**
Add simple POD class to internalize keywords xxxDENT

Dowstream:
https://github.com/OPM/opm-material/pull/404
https://github.com/OPM/opm-simulators/pull/2362

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2362 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2362 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2362 opm-material=404 please

File: opm/parser/eclipse/EclipseState/Tables/DenT.hpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef OPM_PARSER_DENT_HPP
+#define	OPM_PARSER_DENT_HPP
```
Comment (akva2) [inline opm/parser/eclipse/EclipseState/Tables/DenT.hpp]: funky

File: tests/parser/TableManagerTests.cpp [hunk 2]
```diff
@@ -1754,3 +1755,42 @@ REGDIMS
...
+)";
+
+    Opm::Parser parser;
+    const auto& deck = parser.parseString(deck_string);
+    Opm::DenT gd(deck.getKeyword("GASDENT"));
+    Opm::DenT od(deck.getKeyword("OILDENT"));
+    Opm::DenT wd(deck.getKeyword("WATDENT"));
+
+    BOOST_CHECK_EQUAL(gd.size(), 3);
+    BOOST_CHECK( gd == od);
```
Comment (akva2) [inline tests/parser/TableManagerTests.cpp]: wd is unused and will warn.

File: opm/parser/eclipse/EclipseState/Tables/DenT.hpp (hunk 1 above)
Comment (joakim-hove) [reply opm/parser/eclipse/EclipseState/Tables/DenT.hpp]: ok

File: tests/parser/TableManagerTests.cpp (hunk 2 above)
Comment (joakim-hove) [reply tests/parser/TableManagerTests.cpp]: ok

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2362 opm-material=404 please

**Comment by joakim-hove** [comment]
Comments to this or the downstream?

**Comment by atgeirr** [comment]
> Comments to this or the downstream?

Not really. I wonder what these keyword(s) are, briefly, and especially: do they have special behaviour that require attention?

**Comment by akva2** [comment]
see the opm-material PR. it's basically params for thermal pvt setup which are currently read straight from deck, we are working to avoid deck usage there (to remove the need for serialization).

**Comment by atgeirr** [comment]
> see the opm-material PR. it's basically params for thermal pvt setup which are currently read straight from deck, we are working to avoid deck usage there (to remove the need for serialization).

Thanks!

**Pull request merged**